code/modules/projectiles/guns/ego_gun/waw.dm

```diff
@@ -89,31 +89,48 @@
 	Can feathers gain their own wings?"
 	icon_state = "solemnlament"
 	inhand_icon_state = "solemnlament"
+	special = "Firing both solemn lament and solemn vow at the same time will increase damage by 1.5x"
 	ammo_type = /obj/item/ammo_casing/caseless/ego_solemnlament
 	burst_size = 1
 	fire_delay = 0
 	fire_sound = 'sound/abnormalities/funeral/spiritgunwhite.ogg'
 	fire_sound_volume = 30
+	attribute_requirements = list(JUSTICE_ATTRIBUTE = 60)
+
+/obj/item/gun/ego_gun/pistol/solemnlament/before_firing(atom/target, mob/user)
+	for(var/obj/item/gun/ego_gun/pistol/solemnvow/Vow in user.held_items)
+		projectile_damage_multiplier = 1.5
+	..()
+
+/obj/item/gun/ego_gun/pistol/solemnlament/process_fire(atom/target, mob/living/user)
+	..()
+	projectile_damage_multiplier = 1
```

Comment:
guh

Reply:
for some reason initial(projectile_damage_multiplier) was tossing errors.

Reply:
No, I am talking about the way you implemented damage increase.
Can't we check for gun in hands and then pass something to the casing instead?

Reply:
What the fuck

Reply:
That is significantly more snowflakey and shitcodey, and processes extra code every time it fires.
The Projectile Damage multiplier code always fires no matter what, so you're firing off extra lines of code.

I should remind you that projectiles are a major source of lag right now.

Reply:
From what I'm seeing, despite how cursed this may seem, it appears to mirror the funky "Combo" system in terms of how it's been implemented, with a situational damage increase applying before the attack, after which it is removed. Though I'd like to ask if it would work with setting the multiplier in the same proc?